### PR TITLE
Add tests e2e for user settings

### PR DESCRIPTION
**Pull request by MarionGiusti**
New data attributes have been added to access specific columns during testing

### CODE REVIEW

File: tests/test_25_settings_users.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+def add_user_rick(admin_session, browser_session):
+    """Go to Settings User and add user rick."""
+    browser_session.select("#linkSettings").click()
+    browser_session.select("a[href='/settings/users']").click()
+
+    users = [
+        tr.text
+        for tr in browser_session.select_all("#tableUsers tr")
+    ]
+    if "rick" not in users:
```
Comment (pirlgon) [inline tests/test_25_settings_users.py:16]: it's no longer needed as the scope is `module` ?

File: tests/test_25_settings_users.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        browser_session.select("#inputPhone").send_keys("+33611223344")
+        browser_session.select("button.multiselect").click()
+        browser_session.select("input[value='default']").click()
+        browser_session.select("label[for='switchActive']").click()
+        browser_session.select("label[for='switchAdmin']").click()
+        browser_session.select("#submitFormAddUser").click()
+
+
+def test_add_new_user(add_user_rick, browser_session):
+    assert len(browser_session.select_all("#tableUsers tr")) == 3
```
Comment (pirlgon) [inline tests/test_25_settings_users.py:31]: Maybe fragile if we add more users elsewhere?

File: tests/test_25_settings_users.py (hunk 1 above)
Comment (MarionGiusti) [reply tests/test_25_settings_users.py:16]: that's right, thanks !

File: tests/test_25_settings_users.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import pytest
+
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture(scope="module")
+def add_user_rick(admin_session, browser_session):
```
Comment (bersace) [inline tests/test_25_settings_users.py:7]: `user_rick` suffit.

File: tests/test_25_settings_users.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    browser_session.select("#inputPassword2").send_keys("!rick0.@9")
+    browser_session.select("#inputPhone").send_keys("+33611223344")
+    browser_session.select("button.multiselect").click()
+    browser_session.select("input[value='default']").click()
+    browser_session.select("label[for='switchActive']").click()
+    browser_session.select("label[for='switchAdmin']").click()
+    browser_session.select("#submitFormAddUser").click()
+
+
+def test_add_new_user(add_user_rick, browser_session):
```
Comment (bersace) [inline tests/test_25_settings_users.py:25]: `test_create_user`.

**Review by bersace — APPROVED** [review]

**Pull request merged**